mobile/examples/super_resolution/react_native/ort-super-resolution/README.md

```diff
@@ -0,0 +1,125 @@
+
+0. Prepare environment
+    1. install Node.js
+        - Node.js (16.0+): <https://nodejs.org/>
+    2. install expo
+
+        ```shs
+        npm install -g expo-cli
+        ```
+
+    3. install yarn
+
+        ```sh
+        npm install -g yarn
+        ```
+
+1. Setup empty project
+
+   ```sh
+   cd <SOURCE_ROOT>
+   expo init . -t expo-template-blank-typescript
+   yarn
+   ```
+
+   NOTE:
+   - `<SOURCE_ROOT>` refers to the root folder of the source code, where this `README.md` file sits.
+   i.e. `mobile/examples/super_resolution/react_native/ort-super-resolution/`
+
+2. Install onnxruntime-react-native
+
+    ```sh
+    expo install onnxruntime-react-native@dev
+    ```
+
+    NOTE: `@dev` will automatically install the latest dev version of onnxruntime-react-native package. Currently, the latest support is only included in the dev version package.
+    When available, just need to install the latest released version package. for instance, `expo install onnxruntime-react-native@1.15.0`
+
+3. Add your ONNX model to project
+
+    1. Copy the model file to `<SOURCE_ROOT>/assets`.
+
+       In this tutorial, we use `pytorch_superresolution_with_pre_post_processing_opset18.onnx` which is originally from source: [Pytorch Super Resolution](https://pytorch.org/tutorials/advanced/super_resolution_with_onnxruntime.html) and accomodated into [ONNX](https://github.com/onnx/onnx) version with pre/post processing support.
+
+       The model is currently provided under directory `mobile/examples/super_resolution/ios/ORTSuperResolution/ORTSuperResolution/pytorch_superresolution_with_pre_post_processing_opset18.onnx`.
+
+    2. add a new file `metro.config.js` under `<SOURCE_ROOT>` and add the following lines to the file:
+
+       ```js
+       const { getDefaultConfig } = require('@expo/metro-config');
+       const defaultConfig = getDefaultConfig(__dirname);
+       defaultConfig.resolver.assetExts.push('onnx');
+       module.exports = defaultConfig;
+       ```
+
+       This step adds extension `onnx` to the bundler's asset extension list, which allows the bundler to include the model into assets.
+
+4. Setup Android and iOS project.
+
+    We use expo prebuild steps to generate Android/iOS projects folder to consume ONNX Runtime and Ort-Extensions.
+
+    - Use NPM package `onnxruntime-react-native` as an expo plugin. (Recommended way)
+        1. In `<SOURCE_ROOT>/app.json`, add the following line to section `expo`:
+
+           ```
+           "plugins": ["onnxruntime-react-native"],
```

Comment:
this is in app.json already.

may be good to clarify the modifications a user would need to make to a new project, vs. what they need to do to run this example. I assumed these instructions were for running this example

Reply:
yea, I agree. it is included in the configuration already, but thought would still be good to mention.

I will clarify what the user actually needs to edit to run this sample.